src/Main.cpp

```diff
@@ -41,71 +46,164 @@ struct PlayerControlsSystem
 
        
 
-		for (auto&& [entity, bindings, pos, sprite] : access_storage().each())
+		for (auto&& [entity, bindings, pos, sprite] :access_storage().each())
 		{
-			
-
-			if (keys.is_down(bindings.up))
-			{
-				pos.xy.y -= SPEED_MOD * Time::delta_time();
-			}
-
-			if (keys.is_down(bindings.down))
-			{
-				pos.xy.y += SPEED_MOD * Time::delta_time();
-			}
-
-			if (keys.is_down(bindings.left))
-			{
-				side = false;
-				sprite.change_to("fallingHero/runningLeft");
-				pos.xy.x -= SPEED_MOD * Time::delta_time();
-			}
-			else if (keys.is_down(bindings.right))
-			{
-				side = true;
-				sprite.change_to("fallingHero/runningRight");
-				pos.xy.x += SPEED_MOD * Time::delta_time();
-			}
-			else if (keys.is_down(bindings.attack))
-			{
-				if(side == true)
-					sprite.change_to("fallingHero/attack");
-				if(side == false)
-					sprite.change_to("fallingHero/attackLeft");
-			}
-			else{
-				if(side == true)
-					sprite.change_to("fallingHero/heroStanding");
-				if(side == false)
-					sprite.change_to("fallingHero/leftHeroStanding");
+					// if (keys.is_down(bindings.up))
```

Comment:
Indentation is all over the place, please pay attention to this and match the rest of the engine.